src/res/translation/translation_de_DE.ts

```diff
@@ -1497,57 +1497,57 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="186"/>
         <source>Three buffer sizes can be selected</source>
-        <translation type="unfinished"></translation>
+        <translation>Drei Puffergrößen werden unterstützt</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="189"/>
         <source>64 samples: Provides the lowest latency but does not work with all sound cards.</source>
-        <translation type="unfinished"></translation>
+        <translation>64 Samples: Führt zu geringer Verzögerung, funktioniert aber nicht mit allen Soundkarten.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="195"/>
         <source>256 samples: Should only be used when 64 or 128 samples is causing issues.</source>
-        <translation type="unfinished"></translation>
+        <translation>256 Samples: Diese Einstellung sollte nur dann verwendet werden, wenn 64 oder 128 Samples schlechte Ergebnisse liefern.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="207"/>
         <source>If no buffer size is selected and all settings are disabled, this means a buffer size in use by the driver which does not match the values. %1 will still work with this setting but may have restricted performance.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wenn keine Puffergröße ausgewählt ist und alle Einstellungen deaktiviert sind, bedeutet das, dass eine Puffergröße vom Treiber verwendet wird, die nicht den Standardwerten entspricht. %1 wird zwar trotzdem funktionieren, aber eventuell eine schlechtere Leistung bringen.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="213"/>
         <source>The actual buffer delay has influence on the connection, the current upload rate and the overall delay. The lower the buffer size, the higher the probability of a red light in the status indicator (drop outs) and the higher the upload rate and the lower the overall delay.</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Puffergröße hat einen Einfluss auf den Verbindungsstatus, die aktuelle Upload-Rate und die Gesamtverzögerung. Je kleiner der Puffer, desto größer ist die Wahrscheinlichkeit für das Auftreten einer rot leuchtenden LED (was Audioaussetzer anzeigt), eine höhere Upload-Rate und eine niedrigere Gesamtverzögerung.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="270"/>
         <source>Meter Style</source>
-        <translation type="unfinished"></translation>
+        <translation>Audiokanal-Stil</translation>
```

Comment:
Meßanzeige-Stil

Reply:
I think the ß is no longer allowed in the "neue Rechtschreibung"

Reply:
I'm not really happy with this translation. Yes it does measure how loud somebody plays. But I don't really hear "Messanzeige" that often.

Reply:
This sounds like a good improvement. Yes, the spelling should be `Messanzeige` here and in the other places.

Reply:
> I'm not really happy with this translation. Yes it does measure how loud somebody plays. But I don't really hear "Messanzeige" that often.

I agree that it is not common, but it's more accurate than "Audiokanal" (which would translate to "audio channel")), isn't it?

Thinking some more... Maybe `Pegel-Messer`? Does that cover all uses?

Reply:
Yes. I think that's better.
Can you add a suggestion?

Reply:
```suggestion
        <translation>Pegel-Messer-Stil</translation>
```

Reply:
Agree, I'm not happy with Messanzeige-Stil as well... 

I also thought about Pegelmessinstrument, if we really want to have it absolutely in German... 
But , what about VU-Meter -Style ?  I think the musicians will understand.

Reply:
Pegelmesser or Pegel-Messer should work, I think.

Reply:
> I think the ß is no longer allowed in the "neue Rechtschreibung"

My understanding is that it depends on the length of the preceding vowel. https://www.dashoefer.de/thema/wann-schreibt-man-s-wann-ss.html

But I don't know enough to prefer Meß... or Mess... in this context.